src/components/accordion/AccordionButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  Box,
```

Comment:
You can use our Box component we have already extracted instead using the one from chakra-ui/react.